### PR TITLE
feat: add solana network and serum/swap 

### LINKED ISSUE #1722: [Bounty] Solana and Serum in Maskbook
## Introduction and Background

Maskbook is designed and developed as a chain agnostic tool, which relies on no chains and in another word, supports all the chains. Currently, Maskbook is focusing on DAPPs on Ethereum but we would like to integrate more chains into our extension to provide users more options. Therefore, we are redesigning our internal wallet into a multi-chain wallet and by the time the new design is finalized, we would like to start integrating new blockchains other than Ethereum.

### Solana Wallet 

[Solana](https://solana.com/) is a scalable and high performance blockchain. We would like to integrate their wallet creation, management and support relevant on-chain operations. Wallet integration is considered as the **phase two** of this bounty and we are expecting this to start once we release our wallet standard.

### Serum UI Integration

[Serum](https://projectserum.com/) is a decentralized on-chain trading platform which supports both orderbook and swap. We would like to integrate its interface in our [trader plugin](https://github.com/DimensionDev/Maskbook/tree/master/src/plugins/Trader). It allows users to hover their mouse on any token hashtag starting with "$" on Twitter to view its price, trending and a uniswap exchange interface if there is a pair. Integrating Serum, we can support more options other than uniswap so that users could directly trade tokens with Solana without leaving Twitter. This is considered the **phase two**.

## Deadlines

TBD.

### PR DESCRIPTION
**Pull request by developerfred**
### maskbook and serum/solana
close #1722

- [x] add solana wrapper tokens [ref: [@solana/token-list](https://github.com/solana-labs/token-list)] 
- [ ] https://github.com/DimensionDev/Maskbook-Token-List/issues/2
- [ ] add solana/web3js

### CODE REVIEW

**Comment by septs** [comment]
plz update `packages/maskbook/src/_locales/pt/messages.json` file.

**Comment by developerfred** [comment]
> plz update `packages/maskbook/src/_locales/pt/messages.json` file.

done on b6a76947...1dca3004

**Comment by septs** [comment]
@developerfred we use yarn package manager.

plz remove `package-lock.json`

CC @Jack-Works

File: packages/maskbook/src/_locales/pt/messages.json [hunk 1]
```diff
@@ -0,0 +1,366 @@
+{
+    "about": "",
```
Comment (septs) [inline packages/maskbook/src/_locales/pt/messages.json:2]: this field is missing.

**Comment by Jack-Works** [comment]
@developerfred hi I saw you committed a new language (PT) translation. If you split it into another PR, it will be accepted more quickly than pack all things together

File: package.json [hunk 2]
```diff
@@ -62,6 +62,7 @@
     "@msgpack/msgpack": "^1.12.2",
     "@openzeppelin/contracts": "^3.2.0",
     "@servie/events": "^1.0.0",
+    "@solana/web3.js": "^0.87.0",
```
Comment (Jack-Works) [inline package.json:65]: hi @developerfred I'm sorry but you need to rebase this PR (with pnpm) and add this dependency to `packages/maskbook/package.json`

**Comment by yisiliu** [comment]
@developerfred let me know if you still want to work on this and feel free to re-open it.

**Pull request closed without merging**